tests/unit_tests/shape/test_tacs_shape_driver.py

```diff
@@ -188,6 +197,11 @@ def test_shape_and_thick_steady_aeroelastic(self):
             complex_mode=False,
             epsilon=1e-4,
         )
+
+        # for func in f2f_model.get_functions():
```

Comment:
If this is just commented out, can we get rid of it? Same with some previous lines in this file.

Reply:
I removed the active=True arguments like you wanted. Also, removed the functions for loop at the bottom which I temporarily added for debugging this test with the new parallel tacsAIMs. I don't think we need to get rid of the TacsSteadyInterface line necessarily. It shows somebody how you could add that in in the parallel tacsAIM case if you want to.